### PR TITLE
chore: add platform onboarding checklist

### PR DESCRIPTION
**Pull request by hgbarreto**
This is the most recent checklist template used for onboarding to the Platform team

https://github.com/department-of-veterans-affairs/va.gov-team/issues/50615

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+**Platform overview and design**
+- [ ] [Overview of the VA.Gov Platform](https://depo-platform-documentation.scrollhelp.site/getting-started/Welcome.1255276574.html)
+- [ ] [VA.Gov Platform system design and architecture](https://vfs.atlassian.net/wiki/spaces/OT/pages/2354315287/VA.Gov+Platform+Manual+WIP#System-design-and-architecture)
+
+
+### Developer tools 
+- [ ] GitHub
+  - [ ] Join the ['department-of-veterans-affairs' organization ](https://github.com/department-of-veterans-affairs)
+  - [ ] Get access to the [devops repo](https://github.com/department-of-veterans-affairs/devops)
```
Comment (oseasmoran73) [inline .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:51]: Maybe something like: "Get access to the following repos: [vets-website](https://github.com/department-of-veterans-affairs/vets-website), [content-build](https://github.com/department-of-veterans-affairs/content-build/), [devops](https://github.com/department-of-veterans-affairs/devops), [vets-api](https://github.com/department-of-veterans-affairs/vets-api)"


While I don't think all of them will be used, these are the core used by many

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - [ ] [VA.Gov Platform Manual](https://vfs.atlassian.net/wiki/spaces/OT/pages/2354315287/Platform+Infrastructure+Manual+WIP)
+
+**Platform overview and design**
+- [ ] [Overview of the VA.Gov Platform](https://depo-platform-documentation.scrollhelp.site/getting-started/Welcome.1255276574.html)
+- [ ] [VA.Gov Platform system design and architecture](https://vfs.atlassian.net/wiki/spaces/OT/pages/2354315287/VA.Gov+Platform+Manual+WIP#System-design-and-architecture)
+
+
+### Developer tools 
+- [ ] GitHub
+  - [ ] Join the ['department-of-veterans-affairs' organization ](https://github.com/department-of-veterans-affairs)
```
Comment (oseasmoran73) [inline .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:50]: Another task: Might be good to include for something like: "Check with PM to be added specific github-team and COP"

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- [ ] Ensure COR submits a request on your behalf for RA to the VA network 
+
+### Communications 
+
+**Platform coordination**
+- [ ] Share your calendar with VSP teams: va-vsp-team@oddball.io and  vsp-team.group@adhocteam.us
+- [ ] Ensure you're in Google Groups [va-vsp-team](https://groups.google.com/a/oddball.io/g/va-vsp-team) and [va-vsp-engineering](https://groups.google.com/a/oddball.io/g/va-vsp-engineering). 
+- [ ] Use `/support` in #vfs-platform-support to request access.
+- [ ] Ensure you have access to the Google Drive folder ["VSP team"](https://drive.google.com/drive/u/0/folders/0AJ6yVhfytxhFUk9PVA)
+- [ ] DSVA Slack Workspace
```
Comment (oseasmoran73) [inline .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:26]: #vfs-platform-support 
#platform-tech-team-support

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md (hunk 1 above)
Comment (hgbarreto) [reply .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:51]: image

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md (hunk 3 above)
Comment (hgbarreto) [reply .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:28]: image

File: .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md (hunk 2 above)
Comment (hgbarreto) [reply .github/ISSUE_TEMPLATE/platform-onboarding-checklist.md:50]: image
 If there are more GH groups to be added to, maybe I can just insert a 3rd line stating "Check with your PM for any specific GH teams you may need to be added to" ?

**Pull request merged**